src/components/cc-addon-backups/cc-addon-backups.js

```diff
@@ -403,6 +511,10 @@ export class CcAddonBackups extends LitElement {
         cc-input-text {
           --cc-input-font-family: var(--cc-ff-monospace, monospace);
         }
+
+        cc-button[link] * {
+          color: transparent !important;
+        }
```

Comment:
**❔ question:** I'm not sure what this rule is intended for? And not sure if it's actually working? :thinking:

Reply:
Thanks for this @roberttran-cc this rule should have been removed. 

Texts slotted within `cc-button` were visible on hover but it has been fixed with #995 so this is no longer required :innocent: